dbms/src/Flash/Coprocessor/StorageDisaggregatedInterpreter.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Flash/Coprocessor/InterpreterUtils.h>
+#include <Flash/Coprocessor/StorageDisaggregatedInterpreter.h>
+
+
+namespace DB
+{
+void StorageDisaggregatedInterpreter::execute(DAGPipeline & pipeline)
+{
+    auto remote_requests = buildRemoteRequests();
+
+    auto storage = std::make_unique<StorageDisaggregated>(context, table_scan, remote_requests);
+    auto stage = QueryProcessingStage::Enum::FetchColumns;
+    pipeline.streams = storage->read(Names(), SelectQueryInfo(), context, stage, 0, max_streams);
```

Comment:
Insert stream into `context.getDAGContext()->getProfileStreamsMap()` here.

Reply:
Inserted in `StorageDisaggregated::buildReceiverStreams`